[CN-284] Add section for the HotBackup status (#34)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/backup-restore.adoc

----
--

> == Hot Backups Status

Reviewer: ```suggestion
== Checking the Status of a Hot Backup
```

> You can check status of the Hot Backup task by running the folowing command.

Reviewer: ```suggestion
To check the status of a hot backup, run the following command:
```

[source,bash]
----
kubectl get hotbackup hot-backup
----

> You should see the similar output that will indicate that the Hot Backup process finished successfully.

Reviewer: ```suggestion
The status of the hot backup is displayed in the output.
```

What other statuses are there? Can we provide some direction for users if a hot backup fails?

Author: The statuses are self descriptive, and are `"Unknown", "NotStarted", "InProgress", "Failure", "Success"`. Should we provide an additional description for them? For me it seems redundant and will like: `Failure - indicates that the Hot Backup task failed` 😃
